src/ui/settings.ts

```diff
@@ -27,6 +27,8 @@ export interface FullCalendarSettings {
     };
     timeFormat24h: boolean;
     clickToCreateEventFromMonthView: boolean;
+    alwaysOpenInNewTab: boolean;
```

Comment:
```suggestion
    alwaysOpenInNewTab?: boolean;
```

Both of these settings should be optional, since they won't be present in existing Full Calendar installations. Following the type errors should lead you to add some more edge cases to fill in the defaults as they should in those cases.